html/inc/email.inc

```diff
@@ -63,11 +64,16 @@ function send_email($user, $subject, $body, $body_html=null, $email_addr=null) {
         } else if (defined('EMAIL_FROM')) {
             $headers = "From: ". EMAIL_FROM;
         }
-        if ($email_addr) {
-            return mail($email_addr, $subject, $body, $headers);
-        } else {
-            return mail($user->email_addr, $subject, $body, $headers);
+        if (!$email_addr) {
+            $email_addr = $user->email_addr;
+        }
+        if ($body_html) {
+            $body = "<html><body>\n"; 
+            $body = $body_html; 
```

Comment:
You need to append to the string on this line.  At the moment this overwrites <html><body> from the previous line.